rust_src/src/data.rs

```diff
@@ -659,4 +660,33 @@ pub fn bool_vector_subsetp(a: LispObject, b: LispObject) -> LispObject {
     unsafe { bool_vector_binop_driver(a, b, b, BoolVectorOp::BoolVectorSubsetp) }
 }
 
+/// Set SYMBOL's value to NEWVAL, and return NEWVAL.
+#[lisp_fn]
+pub fn set(symbol: LispSymbolRef, newval: LispObject) -> LispObject {
+    unsafe {
+        set_internal(
```

Comment:
You can also use `symbol.into()` here if you prefer. Both do the same task.

Reply:
It looks like `LispObject::from(...)` is used a lot. Is `.into()` considered more idiomatic? I'll leave these ones for now, but I can make a pass over the rest of them later.

Reply:
The `::from` syntax was what we preferred when there was confusion around LispObject in Rust and Lisp_Object in C. Now, either `::from` or `.into()` are fine. The advantage of the `::from` is it really stands out and is a good indicator there is likely more porting to do.